Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -340,8 +343,24 @@ - (void)viewDidLoad {
 
     self.article = [self.dataStore existingArticleWithTitle:self.articleTitle];
     [self fetchArticle];
+
+
+
+[self registerForPreviewingWithDelegate:self sourceView:[self.webViewController.webView wmf_browserView]];
```

Comment:
ideally, we wouldn't need to expose the WebVC's `webView` (or the underlying browser view) publicly—in the interest of keeping WebVC's abstraction from leaking and keeping its responsibilities well encapsulated.  we could instead offer an API that allows users to `registerForBrowserPreviewsWithDelegate:(id<UIViewControllerPreviewing>)`, thereby keeping knowledge of the browser view tucked into the WebVC